### PR TITLE
Added support for dmaker.fan.1c

### PR DESCRIPTION
**Pull request by carlba**
This fan has slightly less features then the more expensive fans. I also added the possibility to hide the fan animation.

### CODE REVIEW

File: fan-xiaomi.js [hunk 1]
```diff
@@ -23,6 +27,20 @@ class FanXiaomi extends HTMLElement {
...
+            this.supportedAttributes.angle = false;
+            this.supportedAttributes.childLock = false;
+            this.supportedAttributes.rotationAngle = false;
+            this.supportedAttributes.speedLevels = 3;
+            this.supportedAttributes.natural_speed = false;
+            this.supportedAttributes.natural_speed_reporting = false;
+        }
+
+        if (attrs['model'] === 'dmaker.fan.p5'){
+            this.supportedAttributes.natural_speed_reporting = false;
```
Comment (insajd) [inline fan-xiaomi.js:40]: p5 fan supports natural_speed, but does not report raw speed. I changed to attributes natrual_speed, and natural_speed_reporting. for other fans, it's true true, for p5 it's true false, for 1c it's false false.

**Review by insajd — COMMENTED** [review]
Thanks for PR! I added fixes for p5 fan. @ikaruswill wanna have a look at these changes with regards to your fan?

**Comment by insajd** [comment]
Also, need to add `show_animation: true` to default config in https://github.com/ikaruswill/lovelace-fan-xiaomi/blob/master/README.md

**Comment by ikaruswill** [comment]
How did I miss this PR? 🤣 Thanks for the PR @carlba and for the tag @insajd I'll take a look in a bit after work.

**Review by insajd — APPROVED** [review]
I have tested this with p5, and it works good. Want to release this as beta

**Pull request merged**